### ISSUE TITLE
Configure failing if no .config file already present

### ISSUE THREAD

**Issue by jlengrand** [RamonGebben/mergify#12]
If I try to configure mergify, I am constantly getting: 

<img width="924" alt="screen shot 2018-04-19 at 9 27 32 am" src="https://user-images.githubusercontent.com/921666/38979633-feac47b6-43ba-11e8-9b35-39f2cddb3ad0.png">

I looked into the `doFetch` class and it looks like both private_token and domain_name are not known to the function. 

If I understand the code correctly, I think I know what is happening
* User runs configure - https://github.com/RamonGebben/mergify/blob/master/index.js#L97
* User enters info
* Infos are verified https://github.com/RamonGebben/mergify/blob/master/index.js#L76
  * Using `doFetch`
  * `doFetch` reads the config https://github.com/RamonGebben/mergify/blob/master/lib/doFetch/index.js#L5
* If verify works, the config is written using `writeConfig` https://github.com/RamonGebben/mergify/blob/master/index.js#L120

Basically, we are trying to read the config before it is written, which explains why I am getting errors.

**Comment by RamonGebben** [comment]
Should be solved in #13
